recipes/jinja2cpp/1.1.0/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8.0)
+project(Jinja2CppTestPackage)
+
+set(CMAKE_CXX_STANDARD 14)
+
+include (${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+CONAN_BASIC_SETUP()
```

Comment:
```suggestion
conan_basic_setup()
```

Reply:
Just for notice: took it from the `poco` test recipe.